esbuild.js

```diff
@@ -3,22 +3,22 @@ const esbuild = require('esbuild');
 const path = require('node:path');
 const os = require('node:os');
 const { sassPlugin } = require('esbuild-sass-plugin');
+const { dtsPlugin } = require('esbuild-plugin-d.ts');
 
 /** @typedef {import('esbuild').BuildOptions} BuildOptions */
 
 const args = process.argv.slice(2);
 const CLIWatch = args.includes('--watch');
-const CLIDevelopment = CLIWatch || args.includes('--development');
 
 /** @type {BuildOptions} */
 const commonConfig = {
     bundle: true,
     target: ['es2020'],
-    minify: !CLIDevelopment,
-    sourcemap: CLIDevelopment,
+    minify: true,
+    sourcemap: true,
```

Comment:
I would prefer to keep the option not to minify. Source maps help, but it can be useful to inspect the generated JS, as well.